### DISCUSSION TITLE
How to Upgrade from Rabbitmq 3.8.9 to the Latest

### DISCUSSION THREAD

**Discussion by mrsethsamuel** [docker-library/rabbitmq#701] [General]
We have a deployment using kubernetes Operator.

How do i upgrade the RabbitMq to the Latest?

I have read and followed the steps here and it still didnt work.

https://www.rabbitmq.com/docs/upgrade

I have also applied the steps here https://www.rabbitmq.com/kubernetes/operator/upgrade-operator

But the rabbitmq server has not updated still

Help please.

**Comment by lukebakken** [comment]
Hello, thanks for using RabbitMQ.

This repository is not the best place to ask this question. I'd like to suggest one of the following:
* https://github.com/rabbitmq/cluster-operator/discussions
* https://groups.google.com/g/rabbitmq-users

Please note that Team RabbitMQ has recently updated its community support policy ([link](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)). You have not provided _nearly_ enough information for anyone to assist you, so if you do ask this question in another place, please be sure to include at least the following:
* Commands you are running to upgrade. You _MUST_ include a transcript that clearly shows the commands you are running and their full output
* Output from Kubernetes and RabbitMQ
* RabbitMQ logs

Saying "it still didnt work" tells us nothing. Thanks.

**Comment by michaelklishin** [comment]
There is an extensive guide on upgrading on rabbitmq.com, which now has search: https://rabbitmq.com/docs/upgrade.

This guide is also mentioned in every release notes document produced in the last several years.
